index.js

```diff
@@ -189,8 +194,13 @@ async function getSize({baseRef, exec, excludes}) {
 			}
 		})
 
+	if (ignoreDeletions) {
+		// remove files that only have deletions to avoid having them in the [includes] property below
+		data = data.filter(d => d.added > 0);
+	}
+
```

Comment:
Not sure about this one tbh. It seems wrong to have modified files that won't be in either `includes` nor `excludes`, but at the same time, if they are deleted files, they won't be counted, so is it ok to not have them in both? or should we move them from `includes` to `excludes`? or just do nothing and leave them in `includes`

Reply:
I guess my question here is whether you want to exclude deleted lines in any file or deleted lines in files deleted by the PR, i.e. just focus on what remains. Either seem plausible to me.

If we want to ignore deleted files, I'd suggest having an `ignores` output. That way we have some signalling to the user that we dropped them during filtering, without it being confused with what by .gitattributes de-selected.

Reply:
I updated the PR to have 2 options. One to ignored deleted lines, one to ignore deleted files. I also added the `ignores` output